fabfile.py

```diff
@@ -65,9 +67,14 @@ def migrate():
 
 
 def backup():
-    run("su %s -c 'mkdir -p %s'" % (env.unprivileged_user, env.backup_dir))
     with cd(env.backup_dir):
         run('%s/bin/python %s/OpenOversight/db_backup.py' % (env.venv_dir, env.code_dir))
         run('mv backup.sql backup.sql_`date +"%d-%m-%Y"`')
-        run('tar czfv backup.tar.gz backup.sql*')
-        get(remote_path="backup.tar.gz", local_path="./backup/backup-%s-%s.tar.gz" % (env.environment, datetime.datetime.now().strftime('%Y%m%d-%H%m%d')))
+        run('su %s -c "aws s3 sync s3://%s /home/nginx/openoversight_backup/s3/%s"'
+            % (env.unprivileged_user, env.s3bucket, env.s3bucket))
+        run('%s/bin/python %s/OpenOversight/db_backup.py' % (env.venv_dir, env.code_dir))
+        run('mv backup.sql backup.sql_`date +"%d-%m-%Y"`')
```

Comment:
Hmm so it looks like these backups are going to keep piling up? Do we have some **not manual** monitoring of disk usage in place?